lib/apimatic-core/logger/sdk_logger.rb

```diff
@@ -39,9 +39,7 @@ def log_response(response)
 
     def apply_log_request_options(request)
       headers_to_log = LoggerHelper.extract_headers_to_log(
-        @log_request.headers_to_include,
-        @log_request.headers_to_exclude,
-        @log_request.headers_to_unmask,
+        @log_request,
```

Comment:
`log_request` and `log_response` should be renamed to `request_logging_config` and `response_logging_config` respectively.